test/mqtt_ascoltatore_spec.js

```diff
@@ -1,3 +1,70 @@
+it("should re-subscribe to topics", function(done) {
```

Comment:
Nothing is run in parallel in the tests, don't worry about it. The only catch is that the test spawn its own broker (https://github.com/mcollina/ascoltatori/blob/master/test/common.js#L98-L104): it is a global, and you can tap into it.